src/types/process.go

```diff
@@ -2,16 +2,26 @@ package types
 
 import (
 	"fmt"
-	"github.com/f1bonacc1/process-compose/src/health"
 	"math"
 	"time"
+
+	"github.com/f1bonacc1/process-compose/src/health"
 )
 
-const DefaultNamespace = "default"
-const PlaceHolderValue = "-"
+const (
+	DefaultNamespace = "default"
+	PlaceHolderValue = "-"
+)
 
 type Processes map[string]ProcessConfig
+
 type Environment []string
+
+type Watch struct {
+	Glob   string   `yaml:"path"`
+	Ignore []string `yaml:"ignore"`
```

Comment:
not *quite* the same as docker-compose's. docker-compose uses just a `Path` where i'm using a `Glob` - i was thinking it might be more convenient to do a glob, but a path might make more sense?